test/OpenTelemetry.Exporter.Geneva.Benchmark/OpenTelemetry.Exporter.Geneva.Benchmark.csproj

```diff
@@ -8,7 +8,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="BenchmarkDotNet" Version="0.13.1" />
```

Comment:
Could use this? https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/215bc921a233200e2acfafc3420e1b763b4ea385/build/Common.nonprod.props#L23